_raml/apps/products/schema/patch_product_productid_customattributes_request_schema.json

```diff
@@ -0,0 +1,11 @@
+{
+  "$schema": "http://json-schema.org/draft-03/schema",
+  "title": "Patch Product ProductId Custom Attributes Request Schema",
+  "type": "object",
+  "properties": {
+    "": {
+      "type": "array of [jsonPatch](page:apps-data-types#jsonpatch)",
+      "description": ""
```

Comment:
No description?

Reply:
@AliceStatler Very often the description is left out. But I agree, let's try to do it consistently. I'll try to think of something.

Reply:
@AliceStatler I've researched a bit in the NOW documentation and I think there is no description because there is no actual property that might be described. It's just a schema that shows how the structure would look like without values. So I think this has to be this way.
@mhoehn Please correct me if I'm wrong ;)

Reply:
@dkluever22 Correct: "It's just a schema that shows how the structure would look like without values."

Reply:
@dkluever22 Okay :) I think it's fine without the description. I was just wondering.